src/game/scenes/Preloader.js

```diff
@@ -42,6 +42,7 @@ export default class Preloader extends Phaser.Scene {
     this.load.image(AnimatronicsNames.MANGLE, `images/heads/${AnimatronicsNames.MANGLE}.png`);
     this.load.image(AnimatronicsNames.GOLDEN_FREDDY, `images/heads/${AnimatronicsNames.GOLDEN_FREDDY}.png`);
     this.load.image(AnimatronicsNames.PUPPET, `images/heads/${AnimatronicsNames.PUPPET}.png`);
+    this.load.image('plank', 'images/other/plank.png')
```

Comment:
made the whole thing with sprite bcs didn't manage to do it other way

Reply:
Making it a sprite is a good idea, but you should make a different complex polygon collider for it. I'll explain the details in Telegram